x-pack/plugins/cloud_security_posture/public/components/chart_panel.tsx

```diff
@@ -64,7 +66,7 @@ export const ChartPanel: React.FC<ChartPanelProps> = ({
         <EuiFlexItem grow={false}>
           {title && (
             <EuiTitle size="xs">
-              <h3 style={{ lineHeight: 'initial' }}>{title}</h3>
+              <h3 style={{ lineHeight: 'initial', paddingLeft: euiTheme.size.s }}>{title}</h3>
```

Comment:
![image](https://user-images.githubusercontent.com/51442161/217871779-6964af34-65f7-41bc-af8b-183c32a16073.png)

looking at the designs, i dont see that the title should be more indented to the right compared to the children. i dont think we need this line, correct me if I'm missing something here